packages/card/card.scss

```diff
@@ -13,15 +16,47 @@ $card-padding-normal: $component-spacing--xl;
     box-shadow: $drop-shadow--medium;
     padding: $card-padding-normal;
     text-align: inherit;
-    transition: box-shadow 0.2s ease, transform 0.2s ease;
 
-    &:hover,
-    &:focus {
+    &__title {
+        margin-bottom: $layout-spacing--small;
+    }
+    &__action {
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        margin-bottom: $layout-spacing--small;
+        margin-top: $layout-spacing--small;
+    }
+    &__media {
+        margin-left: -$component-spacing--xl;
+        margin-right: -$component-spacing--xl;
+        margin-top: -$component-spacing--xl;
+        width: calc(100% + 2 * #{$component-spacing--xl});
+        margin-bottom: $layout-spacing--small;
```

Comment:
Ser at man sikkert bør bytte ut `$component-spacing--xl` med `$card-padding-normal` på disse linjene, så blir de oppdatert hvis man endrer padding på kortet 😊

Du blir også kvitt to hele linjer om du gjør dette:
```scss
margin: $card-padding-normal;
margin-bottom: $layout-spacing--small;
width: calc(100% + 2 * #{$card-padding-normal});
```